deploy/kicbase/Dockerfile

```diff
@@ -191,7 +191,7 @@ EXPOSE 22
 # create docker user for minikube ssh. to match VM using "docker" as username
 RUN adduser --ingroup docker --disabled-password --gecos '' docker 
 RUN adduser docker sudo
-RUN adduser docker podman
+RUN adduser docker podman buildkit
```

Comment:
I think needs to be given twice ?

```
adduser docker podman
adduser docker buildkit
```

Also make sure to create the group.

```
addgroup --system buildkit
```

At the `echo "Installing buildkit ...` block (line 135)